configtx/example_test.go

```diff
@@ -238,12 +240,12 @@ func Example_policies() {
 		panic(err)
 	}
 
-	err = c.RemoveOrdererPolicy(configtx.WritersPolicyKey)
+	err = c.UpdatedConfig().Orderer().RemovePolicy(configtx.WritersPolicyKey)
 	if err != nil {
 		panic(err)
 	}
 
-	err = c.SetOrdererPolicy(configtx.AdminsPolicyKey, "TestPolicy", configtx.Policy{
+	err = c.UpdatedConfig().Orderer().SetPolicy(configtx.AdminsPolicyKey, "TestPolicy", configtx.Policy{
```

Comment:
And here. :) This is an example.

I'll assume this applies to the other occurrences as well.